model/import/Report/ReportBuilder.php

```diff
@@ -34,16 +34,20 @@ public function buildReportsContainer(
         core_kernel_classes_Resource $resource = null
     ): Report {
         $resource = $resource ?? [];
-        if ($importerResults && 0 === $importerResults->getTotalSuccessfulImport()) {
+        if ( ! isset($importerResults) || 0 === $importerResults->getTotalSuccessfulImport()) {
```

Comment:
It is static type or null, no need for isset.

```suggestion
        if ($importerResults || 0 === $importerResults->getTotalSuccessfulImport()) {
```